### ISSUE TITLE
The percentage and average calculation for slider item is broken on mobile app and admin

### ISSUE THREAD

**Issue by natalia-muzyka** [ChildMindInstitute/mindlogger-app#2821]
Two slider items are selected to calculate.
Two slider item has a 0-5 range.

Percentage:

![image.png](https://images.zenhubusercontent.com/60ab57c8916945b978a3da92/49a9639e-cb2d-4d03-b095-7d3519aad00f)

Average:

![image.png](https://images.zenhubusercontent.com/60ab57c8916945b978a3da92/508b9460-f06b-4073-a0af-33754f011cfe)

**Not reproduced on radio and checkbox item.**

Environment:
ML v0.21.48 staging
iPhone 7 // iOS 13.1.1
Google Pixel 5a // Android 11
https://admin-staging.mindlogger.org/
Win 10 / Chrome 103
user-stg@ml.com / 123456
Config report v14 / Test A, Test B
Applet password Qwe123!!!

**Comment by binarybottle** [comment]
Why is the "all" value 0.00 for Test A?

**Comment by natalia-muzyka** [comment]
@binarybottle @WorldImpex Percentage and average score of all types fails because of the slider. I checked radio and checkbox separately, thet work fine together.

**Comment by natalia-muzyka** [comment]
Not reproduced on prod in this applet:
My applet 6 / Double items, Double items (percentage)
test-user1@com.us / qwerty
Applet password Qwe123!!!

**Comment by natalia-muzyka** [comment]
There was an issue because of the incorrect option score value in the slider items on staging activity.
That's why I wasn't able to reproduce it on prod (my comment above).
I fixed option score values and now the average and percentage work for the slider.
Trying to find how I was able to save such invalid scores for slider.

![image.png](https://images.zenhubusercontent.com/60ab57c8916945b978a3da92/7cb2b897-9380-4090-96e0-dd6bfd8b6bc0)

There is another issue with the slider that I was thinking is related to this original ticket. 
If the slider range is started not from 0 there will be a 0 score value for the first response option anyway. Created a new ticket for this:
https://app.zenhub.com/workspaces/mindlogger-5e11094d0c26311588da9626/issues/childmindinstitute/mindlogger-app/2851

cc: @WorldImpex
